### ISSUE TITLE
Can't install v1.2.0

### ISSUE THREAD

**Issue by dblandan** [Unbabel/COMET#98]
## 🐛 Bug
I think v1.2.0 was released, then rolled back as a pre-release. I was able to install it a day or two ago, but this is no longer possible.

### To Reproduce 

```
pip install --upgrade pip  # ensures that pip is current 
pip install unbabel-comet==1.2.0
```

### Expected behaviour

unbabel-comet v1.2.0 is installed

### Screenshots

![image](https://user-images.githubusercontent.com/6475564/205463636-cdc81175-32b7-40d9-acf8-6d3fe6b0292e.png)

### Environment
OS: `Linux 4e9e05fda7ba 5.15.0-56-generic #62-Ubuntu SMP Tue Nov 22 19:54:14 UTC 2022 x86_64 GNU/Linux`
Packaging: `pip 22.3.1 from /usr/local/lib/python3.10/site-packages/pip (python 3.10)`
Version 

### Additional context

Only a day or two ago, I was able to install with poetry & pip.

**Comment by dblandan** [comment]
There's a workaround, of course:

```
pip install -e git+https://github.com/Unbabel/COMET.git@v1.2.0#egg=unbabel-comet
```
but it would be nice if the latest version were in the registry.

**Comment by ricardorei** [comment]
Hi @dblandan I released version 1.2.0 but just for a few hours and then decided to roll-back. We want to release new `default` models but we want to avoid confusion and people reporting scores that are not comparable with previous versions. To avoid that I need to finish documentation on new models first.


Sorry by the confusion! Also, entmax was breaking the install and I asked Ben Peters if they could bump entmax, I see that it was fixed already!
